admin/class-meta-columns.php

```diff
@@ -102,13 +102,21 @@ public function column_content( $column_name, $post_id ) {
 				break;
 			case 'wpseo-metadesc':
 				$metadesc_val = apply_filters( 'wpseo_metadesc', wpseo_replace_vars( WPSEO_Meta::get_value( 'metadesc', $post_id ), get_post( $post_id, ARRAY_A ) ) );
-				$metadesc     = ( '' === $metadesc_val ) ? '<span aria-hidden="true">&#8212;</span><span class="screen-reader-text">' . esc_html__( 'Meta description not set.', 'wordpress-seo' ) . '</span>' : esc_html( $metadesc_val );
-				echo $metadesc;
+				if ( '' === $metadesc_val ) {
+					echo '<span aria-hidden="true">&#8212;</span><span class="screen-reader-text">' . esc_html__( 'Meta description not set.', 'wordpress-seo' ) . '</span>';
+				}
+				else {
```

Comment:
Is it possible to have an early return in the if? If so it might possible to remove the else

Reply:
Yes, that is absolutely possible.
I didn't do that as it makes it slightly less clear what's happening.

Reply:
@andizer Still want this ?

Reply:
Yes, please